algorithms/C/searching/linearsearch.c

```diff
@@ -0,0 +1,29 @@
+/* Program to search data in given array using linear search */
+#include<stdio.h>
+int main()
+{
+    int len;
+    printf("Enter the lenght of array");
+    scanf("%d",&len); //scanning the legth of array
+}
+int arr[len];  //created an array of size 'len'.
```

Comment:
you can't create an array of size len like this because it did work on some compiler so create it dynamically